### ISSUE TITLE
assertion error when train with nerf data

### ISSUE THREAD

**Issue by hanjoonwon** [autonomousvision/sdfstudio#166]
![화면 캡처 2023-07-18 140633](https://github.com/autonomousvision/sdfstudio/assets/125524043/34e7270c-0a7d-4b4c-9b44-8c7e8eb5e322)

I tried to run it according to the command ns-train neus-facto --pipeline.model.sdf-field.inside-outside False --vis viewer --experiment-name neus-facto-fox nerfstudio-data --data data/custom/fox2
( I copied the fox2 folder that I processed to nerfstuio under the data )

**Comment by hanjoonwon** [comment]
> Hi, the current implementation of zip-nerf's inter-level loss is not stable. You could change the loss here: https://github.com/autonomousvision/sdfstudio/blob/master/nerfstudio/models/neus_facto.py#L308 back to `interlevel_loss`.

thank you    
I'm sorry, but I don't think I understand, but what do you want me to change interlevel loss to??

**Comment by niujinshuchong** [comment]
@hanjoonwon Yes, there are two losses implemented https://github.com/autonomousvision/sdfstudio/blob/master/nerfstudio/models/neus_facto.py#L37 and now the default one is interlevel_loss_zip, you could change to interlevel_loss.

**Comment by hanjoonwon** [comment]
> @hanjoonwon Yes, there are two losses implemented https://github.com/autonomousvision/sdfstudio/blob/master/nerfstudio/models/neus_facto.py#L37 and now the default one is interlevel_loss_zip, you could change to interlevel_loss.

i change it   from nerfstudio.model_components.losses import interlevel_loss
and          if self.training:
            loss_dict["interlevel_loss"] = interlevel_loss  
            
but this error occured

![화면 캡처 2023-07-25 150346](https://github.com/autonomousvision/sdfstudio/assets/125524043/fa436653-4613-4d30-83d9-b1a319e1fbf3)

**Comment by niujinshuchong** [comment]
Hi, I guess you need to change it like 

```python
     loss_dict["interlevel_loss"] = self.config.interlevel_loss_mult * interlevel_loss(
                outputs["weights_list"], outputs["ray_samples_list"]
            )
```

**Comment by hanjoonwon** [comment]
> Hi, I guess you need to change it like
> 
> ```python
>      loss_dict["interlevel_loss"] = self.config.interlevel_loss_mult * interlevel_loss(
>                 outputs["weights_list"], outputs["ray_samples_list"]
>             )
> ```


I'm sorry to bother you, but when I finish learning 100%, I get this error, and even if I open the mesh with the mesh lab, the file is weird


![화면 캡처 2023-07-28 183244](https://github.com/autonomousvision/sdfstudio/assets/125524043/c118d7b9-a6da-45c6-9957-6d7f40650797)
